sandbox is not supported in Content-Security-Policy-Report-Only (#27325)
mdn/content: files/en-us/web/http/headers/content-security-policy-report-only/index.md

## Directives

> The directives of the {{HTTPHeader("Content-Security-Policy")}} header can also be applied to `Content-Security-Policy-Report-Only` except for sandbox directive which is unsupported in `Content-Security-Policy-Report-Only`.

Reviewer: ```suggestion
The directives of the {{HTTPHeader("Content-Security-Policy")}} header can also be applied to `Content-Security-Policy-Report-Only`, except for the `sandbox` directive, which is unsupported in `Content-Security-Policy-Report-Only`.
```

> The directives of the {{HTTPHeader("Content-Security-Policy")}} header can also be applied to `Content-Security-Policy-Report-Only`, except for the `sandbox` directive, which is unsupported in `Content-Security-Policy-Report-Only`.

Reviewer: ```suggestion
The directives of the {{HTTPHeader("Content-Security-Policy")}} header can also be applied to `Content-Security-Policy-Report-Only`, except for the `sandbox` directive, which is ignored when used with `Content-Security-Policy-Report-Only`.
```
